### PR TITLE
Added Whats New entry for EarthLocation.of_address

### PR DESCRIPTION
**Pull request by adrn**
Does what it says on the label

cc @eteq

### CODE REVIEW

**Comment by adrn** [comment]
(Canceling builds because this is just text - restart if you disagree!)

**Comment by astrofrog** [comment]
I restarted the Sphinx build, since there are references to check.

**Comment by adrn** [comment]
@astrofrog 👍 thanks

**Comment by adrn** [comment]
I think the supposed failure here is just because of canceled tests?

**Comment by astrofrog** [comment]
@adrn - yep

File: docs/whatsnew/1.3.rst [hunk 1]
```diff
@@ -65,6 +65,14 @@ Coordinate transformations can use JPL Ephemerides
 Earth locations can now be obtained by address
 ----------------------------------------------
 
+With the new :meth:`~astropy.coordinates.EarthLocation.of_address` classmethod,
+:class:`~astropy.coordinates.EarthLocation` objects can now be easily created
+using a string address, for example, "350 5th Ave, New York, NY 10118". This
+works by querying the Google Maps API to retrieve the latitude, longitude, and
+(optional) height of the specified location. This can be useful for quickly
```
Comment (astrofrog) [inline docs/whatsnew/1.3.rst]: Could you actually show the example you mention in 2 lines of code? (it's nice to have some small code examples in the 'what's new'

**Comment by adrn** [comment]
Doh...

**Comment by eteq** [comment]
backported to v1.3.x in b817b70808b52af22c9db5d110fc60296cda873b

**Pull request merged**